uber/model_checks.py

```diff
@@ -53,29 +57,48 @@ def group_paid(group):
 
 def _invalid_phone_number(s):
     if not s.startswith('+'):
-        return len(re.findall(r'\d', s)) != 10 or re.search(SAME_NUMBER_REPEATED, re.sub(r'[^0-9]','',s))
+        return len(re.findall(r'\d', s)) != 10 or re.search(c.SAME_NUMBER_REPEATED, re.sub(r'[^0-9]', '', s))
+
 
 def _invalid_zip_code(s):
     return len(re.findall(r'\d', s)) not in [5, 9]
 
-def attendee_misc(attendee):
-    if attendee.group_id and not attendee.first_name.strip() and not attendee.last_name.strip():
-        return
-
-    if COLLECT_EXACT_BIRTHDATE and attendee.birthdate == '':
-        attendee.birthdate = None # Prevent insertion errors for placeholder attendees
 
-    if not attendee.first_name or not attendee.last_name:
-        return 'First Name and Last Name are required'
-    elif attendee.placeholder:
-        return
-
-    if COLLECT_EXACT_BIRTHDATE and attendee.birthdate is None:
-        return 'Enter your date of birth.'
-    if COLLECT_EXACT_BIRTHDATE and attendee.birthdate > date.today():
-        return 'You cannot be born in the future.'
-
-    if COLLECT_FULL_ADDRESS:
+def ignore_unassigned_and_placeholders(func):
```

Comment:
Yeah, I should at least add a docstring explaining this.  I put this here even instead of in ``uber.decorators`` even though it's a decorator, because this decorator is only used here... though I suppose maybe plugins would want to use it too, in which case it should probably go into decorators.  Or maybe become part of the existing ``@validation.WhateverModel`` decorator, like you could say ``@validation.Attendee(do_not_run_this_validation_for_placeholders_and_unassigned_group_badges=True)`` only less wordy.  Or something along those lines.